cache_utils/decorators.py

```diff
@@ -24,16 +25,35 @@ def cached(timeout, group=None, backend=None, key=None):
     Wrapped callable gets `invalidate` methods. Call `invalidate` with
     same arguments as function and the result for these arguments will be
     invalidated.
+
+    filter_args_kwargs: callable that receives args, kwargs (list, dict)
+                        and returns the new args and kwargs. Those are
+                        combined with function name to get the final key.
+        E.g.
+            def filter_args_kwargs(args, kwargs):
```

Comment:
I had some discussion about `filter_arg_kwargs(args, kwargs)` versus `filter_args_kwargs(*args, **kwargs)` in my other comment, but now I see that this code comment was just outdated and that the actual implementation is like `filter_args_kwargs(*args, **kwargs)` which I think is the better choice. I removed that discussion from my other comment.